examples/computer_vision/detr/README.md

```diff
@@ -0,0 +1,33 @@
+# End-to-end Object Detection with Transformers (Carion et al. 2020)
+This example implements the Detection Transformers (DETR) model introduced by Carion et al. (2020) in their [recent paper](https://arxiv.org/abs/2005.12872).  
+DETR drastically simplifies the object detection pipeline by doing away with complex postprocessing steps required by most other architectures to extract bounding boxes.  
+In place of proposals, anchors, and window centers, DETR directly optimizes a set prediction loss to learn where to focus.  
+
+This implementation largely uses the original implementation provided in [this repo](https://github.com/facebookresearch/detr).  With this Determined implementation, you can easily run DETR with distributed training and hyperparameter search by modifying a few flags in the [experiment config](distributed.yaml).
+
+## Files
+This example is structure as follows:
+* [model_def.py](model_def.py): initializes the data, optimizer, and model and specifies the training and evaluation steps.
+* [model.py](model.py): modifies the original [SetCriterion function](https://github.com/facebookresearch/detr/blob/master/models/detr.py#L83) to use horovod allreduce to sync number of bounding boxes.
+* [data.py](data.py): creates a CocoDetection dataset to work with Google Cloud Storage. 
```

Comment:
Can you create a subsection in `## Files` called `### Configuration Files`, the latter of which has all the associated .yaml?